src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -82,7 +86,11 @@ class SpfxDoctorCommand extends AnonymousCommand {
     '1.0.0': {
       gulp: {
         range: '^3',
-        fix: 'npm i -g gulp@3'
+        fix: 'npm un -g gulp'
+      },
+      gulpCli: {
+        range: '^1 || ^2',
+        fix: 'npm i -g gulp-cli'
```

Comment:
Since we require v1 or v2, we should update the install command, to `npm i -g gulp-cli@2` so that it installs v2 rather than the latest currently available version.

Reply:
Got it. So when gulp-cli@3 comes out (if ever), we would need to update because doctor would report an error.  I'll fix it

Reply:
My understanding is that a specific version of SPFx requires a specific version(s) of gulp-cli, so we should be explicit about them, because in the future new versions might be available that aren't compatible with older versions of SPFx.

Reply:
Fixed in latest commit